pytorch_lightning/metrics/utils.py

```diff
@@ -112,6 +112,29 @@ def _input_format_classification_one_hot(
     return preds.reshape(num_classes, -1), target.reshape(num_classes, -1)
 
 
+def get_mini_groups(idx: torch.Tensor) -> List[torch.Tensor]:
```

Comment:
I think we could find a better name for this function.

Reply:
`get_group_indexes` ?
Remember that is does only `tensor([0, 0, 0, 1, 1, 1, 1]) -> [torch.tensor([0, 1, 2]), tensor([3, 4, 5, 6])]`.

What about moving it as an internal method of the `RetrievalMetric` class?

Reply:
yes, `get_group_indexes` sounds better

Reply:
Changed name